recipes/libpcap/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conans import AutoToolsBuildEnvironment, tools, ConanFile
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class LibPcapConan(ConanFile):
+    name = "libpcap"
+    version = "1.9.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/the-tcpdump-group/libpcap"
+    description = "libpcap is an API for capturing network traffic"
+    license = "BSD-3-Clause"
+    topics = ("networking", "pcap", "sniffing", "network-traffic")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "enable_dbus": [True, False],
+        "enable_bluetooth": [True, False],
+        "enable_usb": [True, False],
+        "enable_packet_ring": [True, False],
+        "disable_universal": [True, False]
+    }
+    default_options = {
+        'shared': False,
+        'enable_dbus': False,
+        'enable_bluetooth': False,
+        'enable_usb': False,
+        'enable_packet_ring': False,
+        'disable_universal': False
+    }
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _is_amd64_to_i386(self):
+        return self.settings.arch == "x86" and tools.detected_architecture() == "x86_64"
+
+    def requirements(self):
+        if self.options.enable_usb:
+            self.requires("libusb/1.0.23")
+
+    def build_requirements(self):
+        if self.settings.os == "Linux":
+            if not tools.which("bison"):
```

Comment:
we usually remove this if, in order to use the same bison version everywhere (same for flex below)